GCRCatalogs/buzzard.py

```diff
@@ -198,12 +203,6 @@ def _subclass_init(self,
                     i -= 1
                     self._quantity_modifiers['Mag_true_{}_des_z01'.format(b)] = (_abs_mask_func, 'truth/AMAG/{}'.format(i))
                     self._quantity_modifiers['mag_true_{}_des'.format(b)] = (_mask_func, 'truth/TMAG/{}'.format(i))
-                    self._quantity_modifiers['mag_{}_des'.format(b)] = (_mask_func, 'truth/OMAG/{}'.format(i))
-                    self._quantity_modifiers['magerr_{}_des'.format(b)] = (_mask_func, 'truth/OMAGERR/{}'.format(i))
```

Comment:
Is the removal of these two lines (201-202) intended?

Reply:
Yes, they were intended. I don't have noisy DES magnitudes in the new version, so these are only read in the old version. I did break backwards compatibility by accident though. The update I just pushed should fix this.